### PR TITLE
GEOPY-598: Support INTEGRATOR DrillholeGroup

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-598 - Support INTEGRATOR DrillholeGroup**

### CODE REVIEW

**Comment by domfournier** [comment]
@sebhmg Not sure what's with pre-commit here. Help please?

File: geoh5py/data/float_data.py [hunk 1]
```diff
@@ -31,8 +31,8 @@ class FloatData(NumericData):
     def primitive_type(cls) -> PrimitiveTypeEnum:
         return PrimitiveTypeEnum.FLOAT
 
-    @classmethod
-    def ndv(cls) -> float:
+    @property
+    def ndv(self) -> float:
```
Comment (sebhmg) [inline geoh5py/data/float_data.py:35]: **suggestion:** decorate with @abstractmethod

File: geoh5py/data/integer_data.py [hunk 2]
```diff
@@ -27,9 +31,44 @@ class IntegerData(NumericData):
     def primitive_type(cls) -> PrimitiveTypeEnum:
         return PrimitiveTypeEnum.INTEGER
 
-    @classmethod
-    def ndv(cls) -> int:
+    @property
+    def ndv(self) -> int:
```
Comment (sebhmg) [inline geoh5py/data/integer_data.py:35]: **suggestion:** decorate with @abstractmethod

File: geoh5py/data/numeric_data.py [hunk 3]
```diff
@@ -33,6 +33,10 @@ class NumericData(Data, ABC):
     def primitive_type(cls) -> PrimitiveTypeEnum:
         return PrimitiveTypeEnum.INVALID
 
+    @property
+    def ndv(self):
```
Comment (sebhmg) [inline geoh5py/data/numeric_data.py:37]: **suggestion:** decorate with @abstractmethod

**Pull request merged**